src/sqs.rs

```diff
@@ -185,24 +188,34 @@ mod test_send_messages_concurrently {
             .queue_url
             .unwrap();
 
-        // FIXME: use batch operations
         let mut results: Vec<usize> = vec![];
         while let Ok(x) = client
             .receive_message()
+            .max_number_of_messages(i32::try_from(MAX_MESSAGES).ok().unwrap())
```

Comment:
You can call `unwrap` on a `Result` directory, so there's no need for the `ok()` here to turn it from a `Result` into an `Option`.

In this case though, I would just use `MAX_MESSAGES as i32`, since we know it's a small constant that wont trigger any truncation or other weird behaviours in practice.